cinema_service/settings.py

```diff
@@ -86,8 +90,11 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "HOST": os.environ["POSTGRES_HOST"],
```

Comment:
env names don't match. Did you run your project?

Reply:
Yes, sorry. I changed an env file with the settings of db in settings.py in the end, but forgot about env.sample